src/test/java/org/broadinstitute/consent/http/service/InstitutionServiceTest.java

```diff
@@ -38,40 +39,126 @@ private void initService() {
     service = new InstitutionService(institutionDAO);
   }
 
+  private Institution initMockModel() {
+    Institution mockInstitution = new Institution();
+    mockInstitution.setName("Test Name");
+    return mockInstitution;
+  }
+
   @Test
-  public void testCreateInstitution() {
+  public void testCreateInstitutionSuccess() {
+    Institution mockInstitution = initMockModel();
+    when(institutionDAO.findInstitutionById(anyInt())).thenReturn(mockInstitution);
     initService();
-    //mock findByInstitution (not insertInstitution) because that is the call that service.createInstitution returns
-    when(institutionDAO.findInstitutionById(anyInt())).thenReturn(getInstitutions().get(0));
-    Institution institute = service.createInstitution(anyString(), anyString(), anyString(), anyInt(), eq(new Date()));
-    assertEquals(getInstitutions().get(0), institute);
+    Institution institution = service.createInstitution(mockInstitution, 1);
+    assertNotNull(institution);
   }
 
   @Test
-  public void testUpdateInstitutionById() {
+  public void testCreateInstitutionBlankName() {
+    Institution mockInstitution = initMockModel();
+    mockInstitution.setName("");
+    initService();
+    try{
+      service.createInstitution(mockInstitution, 1);
+      Assert.fail("Institution CREATE should not succeed with an empty name");
+    } catch(Exception e) {
+      assertEquals("Institution name cannot be null or empty", e.getMessage());
```

Comment:
nit: In general testing, I find that testing message string equality is unnecessary. I prefer to test exception types instead. It's more convenient to add an expect clause in the test annotation so that you know a certain call should throw a certain exception and then you can also simplify the code by removing the try/catch.
Similar feedback in other try/catch/assert blocks.